xml_converter/src/attribute/color.hpp

```diff
@@ -7,18 +7,19 @@
 
 class XMLError;
 namespace waypoint {
-class Color;
+class RGBA;
 }
 
 class Color {
  public:
     std::string hex;
+    float alpha;
```

Comment:
It seems kinda strange to store a `string hex` alongside a `float alpha` here.

Especially because the string hex might be 4 bytes long `#FFFFFFFF` meaning it includes alpha.
Or it might be a non-hex value, such as `red`. Blishhud does not do a great job at describing what valid color values are in their docs unfortunately.

Reply:
This will be addressed in a future pull request. For now, we don't support color.